### PR TITLE
help page added

### PR DESCRIPTION
**Pull request by corinnebosley**
Please let me know if there are more links that I should add.

### CODE REVIEW

File: doc/help.rst [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+`IPython <http://scipy-ipython.readthedocs.io/en/latest/>`_
+
+`Ipywidgets <http://ipywidgets.readthedocs.io/en/latest/>`_
+
+`Iris <http://scitools.org.uk/iris/docs/latest/index.html>`_
+
+`Jupyter <http://jupyter.org/>`_
+
+`Matplotlib <http://matplotlib.org/>`_
```
Comment (dkillick) [inline doc/help.rst]: I wonder if these would be nicer in a list?

Comment (corinnebosley) [inline doc/help.rst]: You mean with bullet points?

File: doc/help.rst [hunk 2]
```diff
@@ -0,0 +1,18 @@
+Useful Links
+============
+
+Cube Browser notebooks make use of a number of Python packages and projects.
+In most cases, you will be able to find help and information from the relevant website.
+Links to the websites of these projects are listed below:
+
+-  `Anaconda <https://anaconda.org/>`_
```
Comment (dkillick) [inline doc/help.rst]: @corinnebosley did you mean this link to point to the Anaconda cloud (i.e the place where Conda users can upload their conda packages to and other users can go to get these packages) or did you mean [Continuum's website for Anaconda](https://www.continuum.io/downloads)?

**Comment by dkillick** [comment]
@corinnebosley other than the query about the Anaconda link I'm :+1:

File: doc/help.rst (hunk 2 above)
Comment (dkillick) [inline doc/help.rst]: Or indeed the Anaconda docs: https://www.continuum.io/documentation

**Comment by corinnebosley** [comment]
@dkillick Changed the link.

**Comment by dkillick** [comment]
I like it :+1:

**Comment by dkillick** [comment]
Yay! My first cube_browser merge! Good stuff @corinnebosley :smiley_cat:

**Pull request merged**